README.md

```diff
@@ -163,61 +179,54 @@ Three sentinel split names are defined for convenience and shared processing:
 
 Additional split names can be used by the user as desired.
 
-```
+```python
 train_split = "train"
 tuning_split = "tuning"
 held_out_split = "held_out"
 
-subject_split = pa.schema(
-    [
-        ("subject_id", pa.int64()),
-        ("split", pa.string()),
-    ]
-)
+
+class SubjectSplit(PyArrowSchema):
+    allow_extra_columns: ClassVar[bool] = False
+
+    subject_id: pa.int64()
+    split: pa.string()
 ```
 
 #### The dataset metadata schema.
 
 ```python
-dataset_metadata_schema = {
-    "type": "object",
-    "properties": {
-        "dataset_name": {"type": "string"},  # The name of the dataset
-        "dataset_version": {"type": "string"},  # The version of the dataset
-        "etl_name": {"type": "string"},  # The name of the ETL process
-        "etl_version": {"type": "string"},  # The version of the ETL process
-        "meds_version": {"type": "string"},  # The version of the MEDS format
-        "created_at": {"type": "string"},  # The creation date in ISO 8601 format
-        "license": {"type": "string"},  # The license of the dataset
-        "location_uri": {"type": "string"},  # The URI of the dataset location
-        "description_uri": {"type": "string"},  # The URI of the dataset description
-        "extension_columns": {
-            "type": "array",
-            "items": {"type": "string"},
-        },  # List of additional columns
-    },
-}
+class DatasetMetadata(JSONSchema):
```

Comment:
As the comments for the exact meaning of these columns is removed, are these documented elsewhere?

Reply:
Great catch! I've added docstrings to all the schema classes with details on each column.